_posts/2024-01-31-release.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Release 2024.05
+author: Priya
+date: 2024-01-31
+category: Jekyll
+layout: post
+---
+
+### Defect Fixes:
+* Resolved an issue where the detailed error information for the Big Query export was not displaying.
+* Resolved an issue where the Data Flow status was sometimes showing incorrect tasks processed.
```

Comment:
Resolved an issue where the Data flow status would sometimes show incorrect number of tasks processed.